docs/index.rst

```diff
@@ -543,8 +515,6 @@ Configuring Login
 
 .. autoclass:: LoginManager
 
-   .. automethod:: setup_app
```

Comment:
Perhaps this can be replaced with `init_app` instead, such that initialization is still documented.